app/types/jsonify.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line import/consistent-type-specifier-style
+import type { Jsonify } from "node_modules/.pnpm/@remix-run+server-runtime@2.4.1_typescript@5.3.3/node_modules/@remix-run/server-runtime/dist/jsonify.d.ts"
```

Comment:
Have you ever tried using import like this ?

```typescript
import type { Jsonify } from "@remix-run/server-runtime/dist/jsonify.d.ts"
```

I tried it in my local and it still works

Reply:
It works and much better. Thanks